src/sentry/static/sentry/app/redux/actions/shared.js

```diff
@@ -0,0 +1,11 @@
+// Helper function that creates an action creator.
+// From: https://redux.js.org/recipes/reducing-boilerplate
+export function ac(type, ...argNames) {
```

Comment:
I don't agree that "ac" should add to the readability over "makeActionCreator". I prefer the longer one. Often when I'm new to a code, or revisit it, I sometimes have a hard time to decipher abbreviations.

Reply:
I agree, and also kind of don't :) In this particular case I feel that the short name was a good choice because `ac` is being used as almost a metaprogramming device, as it were part of the programming language - and only used in the definition of actions. It's similar to the function `t` that's used all over the place in the javascript code for translated strings. So I liked this because when you read the action files, it will be more readable, as this is so easy to read:
```
getCustomer = ac('GET_CUSTOMER', 'name', 'age')
getCustomers = ac('GET_CUSTOMERS') 
```

But, I also thought that if there would be any objections, I would gladly go back to having it the other way. I think I'll change it back in another PR if it's fine with you.

Reply:
I'll change it now, since I have another thing to change.

Reply:
:+1: